cinch/roles/jenkins_slave/tasks/main.yml

```diff
@@ -36,6 +37,14 @@
     dest: "{{ jenkins_user_home }}/{{ jswarm_filename }}"
     url: "{{ jswarm_download }}"
 
+- name: set variable for docker container
+  set_fact:
+    jenkins_master_url: "{{ '{% raw %}{{ .Env.JENKINS_MASTER_URL }}{% endraw %}' }}"
```

Comment:
I know that this templating stuff was tricky to get working.  This might deserve a comment just to explain why it was required and what the expected end result should be.